app/utils/callback/lava.py

```diff
@@ -19,6 +19,8 @@
 import json
 import urllib2
 
+import dateutil
```

Comment:
This standard import should be in the list above, between `codecs` and `errno`.

Reply:
I added it in a separate line as `dateutil` is not a standard library module. Should that be mixed together with the others?

Reply:
I would put it in the list at the top, to only differentiate local modules in this project from the ones installed on the system.  If it's a new requirement then it should probably be added to `requirements.txt` though, but I see that `dateutil` is already used in `lava_log_parser.py`.

Reply:
... and `python-dateutil` is already listed in the `requirements.txt`.